src/dat.c

```diff
@@ -1,5 +1,14 @@
 #include "erfa.h"
 
+static eraLEAPSECOND *changes = 0;
+static int NDAT = 0;
+
+void eraUpdateLeapSeconds(eraLEAPSECOND *leapseconds, int count)
+{
+   changes = leapseconds;
+   NDAT = count;
+}
+
```

Comment:
`eraUpdateLeapSeconds` can now be used to change the leap seconds array `changes` and size `NDAT`.